src/components/scene/inspector.js

```diff
@@ -4,7 +4,9 @@ var bind = require('../../utils/bind');
 var pkg = require('../../../package');
 var registerComponent = require('../../core/component').registerComponent;
 
-var INSPECTOR_URL = pkg.homepage + 'releases/' + pkg.version + '/aframe-inspector.min.js';
+var INSPECTOR_MASTER_URL = 'https://aframe.io/aframe-inspector/build/aframe-inspector.js';
+var INSPECTOR_RELEASE_URL = pkg.homepage + 'releases/' + pkg.version + '/aframe-inspector.min.js';
+var INSPECTOR_URL = process.env.BUILD_TYPE === 'development' ? INSPECTOR_MASTER_URL : INSPECTOR_RELEASE_URL;
```

Comment:
We should name the environment variable for what it does, pointing to Inspector dev builds. It doesn't have any effect on the overall build type of A-Frame.